src/popup/Styles/popup.scss

```diff
@@ -115,7 +115,7 @@ html {
     -moz-box-sizing: border-box;
 }
 
-#popup-container [role='heading'] {
+#popup-container :is(h1, h2, h3, h4, h5, h6) {
     width: fit-content;
     padding-left: 0;
 }
```

Comment:
The newly updated popup heading could use a `margin-block: 0;` to remove some extra spacing🙂

Reply:
Thanks a lot for the suggestions, I have implemented both however the e2e report is still failing when I run it locally. I am struggeling to find the source of the issue, and in my digging I found another '<div role=heading' element being used in the rule-details-group in the DetailsView which I believe comes from the rules-with-instance.tsx component. However I am not sure how to change this to a native element. This also doesn't make sense as the source of the failing tests as this file hasn't been changed for a long time. So long story short, I am a bit lost again, but will have a look again tomorrow -hopefully with fresh eyes

Reply:
Thank you for looking into that! For the report e2e test, I'm still able to get that working with `yarn build:package:report` and then `yarn test:report:e2e -- -- -- -u`, and the snapshot updates look reasonable to me. 

For the remaining role=heading `div`, now that you mention it, I'm seeing a few other role=heading based headings that were a bit trickier to find in the code. Sorry for not finding those earlier! I've found [one with the props defined elsewhere](https://github.com/microsoft/accessibility-insights-web/blob/63a0325b99debe7f2a81aa8294f3b78efb342b87/src/common/components/cards/collapsible-component-cards.tsx#L79) (probably the one you're seeing that gets headingLevel from rules-with-instance.tsx), and then a few using `span` rather than `div`: [1](https://github.com/microsoft/accessibility-insights-web/blob/63a0325b99debe7f2a81aa8294f3b78efb342b87/src/DetailsView/components/adhoc-tab-stops-test-view.tsx#L166), [2](https://github.com/microsoft/accessibility-insights-web/blob/63a0325b99debe7f2a81aa8294f3b78efb342b87/src/DetailsView/components/requirement-instructions.tsx#L18), [3](https://github.com/microsoft/accessibility-insights-web/blob/63a0325b99debe7f2a81aa8294f3b78efb342b87/src/electron/views/tab-stops/tab-stops-testing-content.tsx#L47).

Given how much you've already put into this already, it would be totally fair to wrap up this PR without adding any of those, or just addressing that last `div` heading. Whatever you'd like to do!

Reply:
I am looking into it and will get back - not sure if I will be able to fix it all so might make sense to scope some out but will have a closer look first